.gitignore

```diff
@@ -2,6 +2,7 @@
 **/lib/**
 **/etc/**
 *.zip
+node_modules
```

Comment:
I mean I don't have a problem with it, but a better idea would be to clean your project directory and pay attention to what you add to the repo.